src/videojs-contrib-hls.js

```diff
@@ -48,6 +48,25 @@ Object.defineProperty(Hls, 'GOAL_BUFFER_LENGTH', {
   }
 });
 
+Object.defineProperty(Hls, 'BUFFER_LOW_WATER_LINE', {
+  get() {
+    videojs.log.warn('using Hls.BUFFER_LOW_WATER_LINE is UNSAFE be sure ' +
```

Comment:
I just copy pasted the getter/setter for `GOAL_BUFFER_LENGTH` and changed the property to `BUFFER_LOW_WATER_LINE`